templates/student/index.html.twig

```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Hello StudentController!{% endblock %}
```

Comment:
Nepaliekame šiukšlių.

Kolegoms reikės visą tavo kodą skaityti.

Plačiau:
 * https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews